tools/scripts/create-playground.js

```diff
@@ -0,0 +1,45 @@
+const {
+  cleanup,
+  runYarnInstall,
+  tmpProjPath,
+} = require('@nrwl/nx-plugin/testing');
+const { readWorkspaceJson } = require('@nrwl/workspace');
+const { appRootPath } = require('@nrwl/workspace/src/utils/app-root');
+const { execSync } = require('child_process');
+const { ensureDirSync, readFileSync, writeFileSync } = require('fs-extra');
+const { getPublishableLibNames } = require('./utils');
+
+console.log('\nCreating playground. This may take a few minutes.');
+
+const workspaceJson = readWorkspaceJson();
+const publishableLibNames = getPublishableLibNames(workspaceJson);
+
+execSync(
+  `yarn nx run-many --target build --projects ${publishableLibNames.join()}`
+);
+
+ensureDirSync(tmpProjPath());
+
+cleanup();
+
+execSync(
+  `node ${require.resolve(
+    '@nrwl/tao'
+  )} new proj --no-interactive --skip-install --collection=@nrwl/workspace --npmScope=proj`,
```

Comment:
Using `playground` instead of `proj` would allow both e2e and playground to exist simultaneously rather than overwriting one another.

Reply:
I opted to generate the playground in a separate directory. I prefer the semantics of moving the playground outside of the e2e directory because it's not related to e2e tests.